ansible/vars/dut_droplet_vlans-1.yml

```diff
@@ -0,0 +1,11 @@
+Vlans:
+  Vlan100: 
+   id: 100
+   intfs: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
+   tag: 100
+   subnets: 192.168.100.0/21
```

Comment:
Also I'd suggest to rename subnets to prefix

Reply:
Ack

Reply:
I don't see it is done

Reply:
this file is part of Option A. will take it out